python/cusignal/radartools/radartools.py

```diff
@@ -172,7 +172,7 @@ def ambgfun(x, fs, prf, y=None, cut='2d', cutValue=0):
         for col in range(v.shape[1]):
             v_shift[:, col] = cp.roll(v[:, col], -col, axis=0)
```

Comment:
This command is striping the imaginary part. Are you sure you mean to do that???
If yes, there's no need for the `cp.conj(v_shift)` below.

Reply:
Yeah -- I was testing this with real valued input data, but that won't always be the case. Basically, the v_shift `zeros` matrix should be of complex type.